### DISCUSSION TITLE
Error in Web Interface in new version Tasmota 14.3.0.7 with WS2812

### DISCUSSION THREAD

**Discussion by Romik4x4** [arendst/Tasmota#22580] [Q&A]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._

The latest version has problems with the control output for WS2812.

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ ] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [x] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

ver 14.3.0.7
![err1](https://github.com/user-attachments/assets/f75a8f8f-70fb-443e-a7c5-153ea1982e33)
ver 14.0.0.4
![err2](https://github.com/user-attachments/assets/2c4370a3-0c74-4a0f-93d7-a846a23fe5a1)

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment]
The user interface in WebUI has changed. You did not tell what you are missing.

**Comment by Romik4x4** [comment]
No Button with my relay via MCP230xx R1 R2 R3 R4 and no on/off display via TM1637 button.
